_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -46766,7 +46766,7 @@
 /obj/docking_port/stationary{
 	dir = 2;
 	dwidth = 11;
-	height = 22;
+	height = 24;
```

Comment:
If this needs to be a standard size in every map or it stops working, maybe it should just be a subtype? Or are the width/height swapped based on its direction?

Reply:
subtypes are always optimal but this can also be addressed at a later date imo